package.json

```diff
@@ -40,6 +40,10 @@
 		"docker:ngrok": "yarn docker:compose -f docker/docker-compose-ngrok.yml",
 		"docker:ngrok-up": "yarn docker:ngrok up",
 		"docker:ngrok-down": "yarn docker:ngrok down",
+		"docker:jt": "echo 'The tool is only available to Automatticians, see `docker/README.md` for more information.",
```

Comment:
The "docker:jt" command seems kind of pointless now, because none of the rest of the messages refer to it.

Reply:
Good point, no need for that command now.